game/gbtd_gbr0.ksy

```diff
@@ -0,0 +1,300 @@
+        0x0004: tile_export
+        0x0005: tile_import
+        0x000D: palettes
+        0x000E: tile_pal
+        0x00FF: deleted
+
+    types:
+      producer:
+        seq:
+          - id: name
+            type: strz
+            size: 30
+          - id: version
+            type: strz
+            size: 10
+          - id: info
+            type: strz
+            size: 80
+
+      tile_data:
+        seq:
+          - id: name
+            type: strz
+            size: 30
+          - id: width
+            type: u2
+          - id: height
+            type: u2
+          - id: count
+            type: u2
+          - id: color_set
+            size: 4
+          - id: tiles
+            size: width*height
+            repeat: expr
+            repeat-expr: count
+
+      tile_settings:
+        enums:
+          split_order:
+            0: left2right_top2bottom
+            1: top2bottom_left2right
+
+          color_set:
+            0: pocket
+            1: gameboy
+            2: gbc
+            3: sgb
+
+        types:
+          display_settings:
+            seq:
+              - id: grid_shown
+                type: b1
+              - id: nibble_markers_shown
+                type: b1
+
+        seq:
+          - id: tile_id
+            type: u2
+          - id: simple
+            type: u1
+          - id: flags
+            type: display_settings
+          - id: left_color
+            type: u1
+          - id: right_color
+            type: u1
+          - id: split_width
+            type: u2
+          - id: split_height
+            type: u2
+          - id: split_order
+            type: u1 # Note: Official specification incorrectly says "long"
+            enum: split_order
+          - id: color_set
+            type: u1
+            enum: color_set
+          - id: bookmarks
+            type: u2
+            repeat: expr
+            repeat-expr: 3
+          - id: auto_update
+            type: u1
+        
+      tile_export:
+        enums:
+          compression:
+            0x00: none
+            0x01: gb_compress
+
+          format:
+            0x00: gameboy_4_color
+            0x01: gameboy_2_color
+            0x03: byte_per_color
+
+          counter_type:
+            0x00: none
+            0x01: count_byte_as_byte
+            0x02: count_byte_as_word
+            0x03: count_byte_as_constant
+            0x04: count_tile_as_byte
+            0x05: count_tile_as_word
+            0x06: count_tile_as_constant
+            0x07: count_8x8_as_byte
+            0x08: count_8x8_as_word
+            0x09: count_8x8_as_constant
+
+          palettes:
+            0: none
+            1: constant_per_entry
+            2: bits_2_per_entry
+            3: bits_4_per_entry
+            4: byte_1_per_entry
+
+        seq:
+          - id: tile_id
+            type: u2
+          - id: file_name
+            type: strz
+            size: 128
+          - id: file_type
+            type: u1
```

Comment:
Is it also a enum? Also is it the same `file_type` as in `tile_import`? Is it a substructure then?

Reply:
It is an enum but it is not the same as tile_import. Will change to enum.